cli/2024-06-04-aws-irsa-support.md

```diff
@@ -0,0 +1,326 @@
+# Radius AWS Workload Identity Support
+
+* **Status**: In Review
+* **Author**: Nithya Subramanian (@nithyatsu)
+
+## Overview
+
+A software workload such as a container-based application, service or script needs an identity to authenticate, access, and communicate with services that are distributed across different platforms and/or cloud providers. Radius uses AWS IAM (Identity and Access Management) credentials to deploy and access AWS resources today:
```

Comment:
```suggestion
A software workload such as a container-based application, service or script needs an identity to authenticate, access, and communicate with services that are distributed across different platforms and/or cloud providers. Radius uses static credentials (Access Key) to interact with AWS IAM (Identity and Access Management) so that we can deploy and access AWS resources today:
```